src/main/kotlin/lotto/LottoMachine.kt

```diff
@@ -0,0 +1,75 @@
+package lotto
+
+object LottoMachine {
+    fun start() {
+        val amount = InputView.inputPurchaseAmount()
+        val ticketList = generateTickets(amount)
+        val winningNumbers = InputView.inputWinningNumbers()
+        val bonusNumber = InputView.inputBonusNumber(winningNumbers)
+        calculateStats(ticketList, winningNumbers, bonusNumber)
+        OutputView.displayResults(winStats, calculateReturnRate(amount))
+    }
+
+    fun generateTickets(amount: Int): List<Ticket> {
+        val ticketCount = amount / Const.PRICE
+        val ticketsList = mutableListOf<Ticket>()
```

Comment:
How about creating a `Tickets` value object and sending it a message to generate tickets?